cmd/check-gh-automation/main.go

```diff
@@ -164,10 +183,39 @@ func checkRepos(repos []string, bots []string, ignore sets.Set[string], client a
 			repoLogger.Info("all bots are org members or repo collaborators")
 		}
 
+		if sets.NewString(pluginsForRepo.Plugins...).Has("cherrypick") {
+			isMember, err := client.IsMember(org, "openshift-cherrypick-robot")
+			if err != nil {
+				return nil, fmt.Errorf("unable to determine if openshift-cherrypick-robot is a member of %s: %w", org, err)
+			}
+			if !isMember {
+				failing.Insert(orgRepo)
+				repoLogger.Error("openshift-cherrypick-robot is not an org member or repo collaborator")
+			}
+		}
+
 		appInstalled, err := client.IsAppInstalled(org, repo)
+
 		if err != nil {
 			return nil, fmt.Errorf("unable to determine if openshift-ci app is installed on %s/%s: %w", org, repo, err)
 		}
+
+		externalPlugins := configAgent.Config().ExternalPlugins[orgRepo]
+
+		for _, plugin := range externalPlugins {
+			if plugin.Name == "cherrypick" {
+				isMember, err := client.IsMember(org, "openshift-cherrypick-robot")
```

Comment:
why are we checking for org membership twice?

Reply:
removed this section